content/events/2020-02-11-how-to-talk.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "How to talk really good"
+type: event
+date: 2020-02-11
+starttime: 18:00
+endtime: 21:00
+presenter: Jessica White & Moreton Brockley
+presenter_img: https://pbs.twimg.com/profile_images/1045414356983271424/hAooD0u6_400x400.jpg
+presenter_bio: |
+    Jessica is a developer, international speaker & avid learner. She is currently working for Experian and has spoken about numerous topics including community building, monitoring and continuous improvement of systems.
+
+    Outside of work, she is a co-founder and co-organiser of DDD East Midlands Conference. Previous to this she has founded or been involved in the organisation and coordination of other events. She also mentors and helps other speakers. 
+
+    She is easily bribed with coffee & loves hearing about other people’s experiences.
+
+    <----------------------------------->
```

Comment:
The presenter bio supports markdown - perhaps just `---` is better here for a `<hr />`?

Reply:
Aha! I didn't know that, gonna change it 🙂